src/frontends/tensorflow_lite/src/input_model.hpp

```diff
@@ -24,6 +24,9 @@ class InputModel : public ov::frontend::InputModel {
     std::map<std::string, std::shared_ptr<ov::frontend::tensorflow_lite::TensorLitePlace>> get_tensor_places() const;
     std::map<std::string, Output<Node>> get_tensor_values() const;
 
+    ////// Subgraph Handling /////
```

Comment:
strange comment:)

Reply:
I struggle to understand the action which is required here. Could you make your comments structural to make this review productive?